components/primary_button/README.md

```diff
@@ -35,22 +35,15 @@ The rounded edges on the button help differentiate it from other layout items on
 1.	Button label should appear underlined on hover
 
 ## Accessibility
-This button has been built according to [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/) standards and all government services should strive to meet this level.  This component successfully includes the following accessibility features:
-
-### Screenreaders
-* Proper ALT labels are included
-* Language tag has been set to english
+In addition to [fundamental accessibility]() requirements, this component meets or exceeds the [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/) standards by incorporating the following criteria.
 
 ### Colour Contrast
-* Contrast ratios are at least 7:1 for normal text and 4.5:1 for large text
-
-### Keyboard Navigation
-* Supports tab navigation for use without a mouse
+* Contrast ratios are at least 7:1 for normal text and 4.5:1 for large text when in both hover and static states
```

Comment:
It's not clear what is meant by normal text vs large text. Isn't this just stating the contrast ratio for either normal state vs hover state?

Reply:
@dlevineBC I put in the description just for normal text since that should be the only text in a button. This is specifically stating that the text retains sufficient colour contrast when the hover state kicks in.